docker/android/Dockerfile

```diff
@@ -1,4 +1,7 @@
-FROM ubuntu:zesty
+# sys_image can be overridden from the CLI
+ARG sys_image=debian:jessie
+
```

Comment:
Probably yes ... however I am very reluctant to update the current Android toolchain, it took quite a lot of work to get the full build running especially the integration with the emulators.
The NDK upgrade would be fine if it has no impact on the rest of the Android SDK setup, because most of those used versions are fine tuned to get the Android emulators working inside Docker containers.
I will put this point on my roadmap and let you know once I figured out if this can be done without side-effects.